packages/strapi-plugin-content-manager/admin/src/components/Group/Form.js

```diff
@@ -0,0 +1,50 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import Inputs from '../Inputs';
+import SelectWrapper from '../SelectWrapper';
+
+const Form = ({ keys, layout, modifiedData, fieldName, onChange }) => {
+  const currentField = get(layout, ['schema', 'attributes', fieldName], '');
```

Comment:
This value is constant throughout the lifecycle of the component so it should be changed to 
`const currentField = useMemo(() => get(layout, ['schema', 'attributes', fieldName], ''))`;